tests/testthat/test-generate_variability_terms.R

```diff
@@ -62,6 +62,24 @@ test_that("Generates RUV error for multiple individuals", {
   expect_true(abs(mean(out$add)) < 0.01)
 })
 
+test_that("Adds RUV to both ltbs models correctly", {
+  out <- generate_ruv(
+    ids = 1:30,
+    n_iter = 40,
+    tdm_sample_time = seq(2, 100, 2),
+    prop = 0.000,
+    add = 0.25,
+    ltbs=T,
```

Comment:
style guide prefers:

```suggestion
    ltbs = TRUE,
```